2-mandatory/1-create-functions.js

```diff
@@ -3,15 +3,18 @@ Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the first five elements of the passed array.
 */
-function first5() {
+function first5(arr) {
+  return arr.slice(0,5)
 }
 
 /*
 Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the same elements, except sorted.
 */
-function sortArray() {
+function sortArray(arr) {
+  let numbersCopy = [...arr]
+  return numbersCopy.sort()
```

Comment:
you can return directly the sorted array without making a copy.
`return arr.sort()`

Reply:
thank you did it with one line and .slice method
return arr.slice().sort()